MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -356,8 +358,70 @@ MeshLib::Mesh* VtkMeshConverter::convertUnstructuredGrid(vtkUnstructuredGrid* gr
 		elements[i] = elem;
 	}
 
-	return new MeshLib::Mesh(mesh_name, nodes, elements);
+	MeshLib::Mesh* mesh = new MeshLib::Mesh(mesh_name, nodes, elements);
+	convertScalarArrays(*grid, *mesh);
+
+	return mesh;
+}
+
+void VtkMeshConverter::convertScalarArrays(vtkUnstructuredGrid &grid, MeshLib::Mesh &mesh)
+{
+	vtkPointData* point_data = grid.GetPointData();
+	if (point_data != nullptr)
+	{
+		unsigned const n_point_arrays = static_cast<unsigned>(point_data->GetNumberOfArrays());
+		for (unsigned i=0; i<n_point_arrays; ++i)
+			convertArray(point_data->GetArray(i), mesh.getProperties(), MeshLib::MeshItemType::Node);
+	}
+
+	vtkCellData* cell_data = grid.GetCellData();
+	if (cell_data == nullptr)
```

Comment:
i suppose it doesn't hurt to have this test here, but if ```cell_data == nullptr``` the converter should not have gotten to this point ...